docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+.. _orion:
```

Comment:
What is the final solution regarding the translation? Depending on it I'd add a point here as well.

Reply:
I did not yet find time to look into it, I will add the required documentation afterwards.

Reply:
Sounds good.